src/gui/Src/Gui/HandlesView.cpp

```diff
@@ -118,6 +119,8 @@ HandlesView::HandlesView(QWidget* parent) : QWidget(parent)
     mActionFollowProc = new QAction(DIcon(ArchValue("processor32", "processor64")), tr("Follow Proc in Disassembler"), this);
     connect(mActionFollowProc, SIGNAL(triggered()), this, SLOT(followInDisasmSlot()));
     mActionFollowProc->setShortcut(Qt::Key_Return);
+    mActionFollowThread = new QAction(DIcon("arrow-threads"), tr("Follow Thread in Threads window"), this);

```

Comment:
I think it should be named `Follow in Threads` to stay consistent with the call stack:

![image](https://github.com/x64dbg/x64dbg/assets/2458265/d045f027-ade5-4a3e-b140-00e82e6ee838)

Reply:
Added icons for CallStackView and adjusted the names